### PR TITLE
[Evo] Ref #5190 - send email notification to the access requestor

### PR DESCRIPTION
**Pull request by nabdali**
- Add starter mail dependencies
- Create sample email service
- Send email notification to requestor
- Add sample email configuration in config file

### CODE REVIEW

File: src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt [hunk 1]
```diff
@@ -14,10 +15,12 @@ import org.springframework.web.reactive.function.server.ServerResponse
 import org.springframework.web.reactive.function.server.ServerResponse.*
 import org.springframework.web.reactive.function.server.bodyToMono
 import reactor.core.publisher.Mono
+import javax.mail.SendFailedException
 
 @Component
 class DataAccessRequestHandler(
-    private val dataAccessRequestRepository: DataAccessRequestRepository
+    private val dataAccessRequestRepository: DataAccessRequestRepository,
+    private val emailService: EmailService
```
Comment (nabdali) [inline src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt:23]: I think is beter to use this service in a data access request service when there's going to be one

Comment (bobeal) [reply src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt:23]: I don't understand what you meant here

File: src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt [hunk 2]
```diff
@@ -79,6 +82,16 @@ class DataAccessRequestHandler(
                                 fields = currentDataAccessRequest.fields
                             )
                         ).subscribe()
+
+                        try {
```
Comment (bobeal) [inline src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt]: just add a TODO (or file an issue) for a later decoupling of the mail sending feature (but ok for now)

File: src/main/kotlin/org/ozwillo/dcimporter/service/EmailService.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+@Service
+class EmailService(@Autowired private var emailSender: JavaMailSender? = null) {
+
+    fun sendSimpleMessage(to: String, subject: String, text: String) {
+        val message = SimpleMailMessage()
+        message.setTo(to)
+        message.setSubject(subject)
+        message.setText(text)
+        emailSender!!.send(message)
```
Comment (bobeal) [inline src/main/kotlin/org/ozwillo/dcimporter/service/EmailService.kt]: can't it be cleaner (ie no !! enforcing) ?

File: src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt (hunk 1 above)
Comment (nabdali) [reply src/main/kotlin/org/ozwillo/dcimporter/web/DataAccessRequestHandler.kt:23]: Sorry for the typo, I meant that we need to create a service for the data access request

**Pull request merged**